### PR TITLE
fix some refs/ remove eric from acknowl now that an author

### PR DESCRIPTION
**Pull request by jharenza**
Noticed during the handy compared docs change that a few refs were not displaying properly, so testing updates here. Also removed Eric from acknowledgements as he is now an author.

### CODE REVIEW

File: content/03.results.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ These transcriptomic scores can be utilized to infer _TP53_ function in the abse
 
 We used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (`EXTEND`) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; @doi:10.1093/carcin/bgp268], such that higher `EXTEND` scores suggest increased malignant potential.
 While we did not find that tumors with _TERT_ promoter (TERTp) mutations (N = 6) had significantly higher telomerase activity scores than non-mutated tumors (Wilcoxon p-value = 0.1196), we observed that `EXTEND` scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).
-Since catalytically-active telomerase requires a combination of full-length _TERT_, _TERC_, as well as accessory proteins [@pmid:9751630], we expect that `EXTEND` scores may not be exclusively correlated with _TERT_ alterations and expression.
+Since catalytically-active telomerase requires a combination of full-length _TERT_, _TERC_, as well as accessory proteins [@pubmed:9751630], we expect that `EXTEND` scores may not be exclusively correlated with _TERT_ alterations and expression.
```
Comment (jharenza) [inline content/03.results.md:220]: @jaclyn-taroni @jashapiro I can't get this one to render - it seems like pmid no longer works, so I had to change all over to `pubmed` - weird.

This one has no doi, no pmc. pubmed doesn't work and https doesn't work. @jashapiro I know you did some manual refs before - not sure if you can help with this one?

Comment (jaclyn-taroni) [reply content/03.results.md:220]: I don't think this should be considered a blocker (so long as we are fully aware of what this _should be_) given our deadline and what else we have going on on our side.

Comment (jharenza) [reply content/03.results.md:220]: Yeah I can manually insert- didn't know if the manual ref was a quick thing or not.

Comment (jharenza) [reply content/03.results.md:220]: Although, and I have to check, if it's not numbered or placeholder numbered - I would have to renumber everything following (which I'm not going to do manually), I will just push it at the end as a manual ref in the doc.

Comment (jaclyn-taroni) [reply content/03.results.md:220]: My thought process is that if this were to be accepted, there would almost certainly be author queries related to citations that we aren't even aware of yet!

**Comment by jharenza** [comment]
@jaclyn-taroni once this is complete, the ref should be fixed! Then, I think we are good to merge if you approve!

File: content/03.results.md [hunk 2]
```diff
@@ -283,7 +283,7 @@ This pattern suggests these subtypes may be driven by common transcriptional pro
 
 We next performed gene set variant analysis (GSVA) for Hallmark cancer gene sets to demonstrate activation of underlying oncogenic pathways (**Figure {@fig:Fig5}B** and quantified immune cell fractions across OpenPBTA tumors using quanTIseq (**Figure {@fig:Fig5}C** and **Figure {@fig:S6}E**).
 Through these analyses, we were able to recapitulate previously-described tumor biology.
-For example, HGG, DMG, MB, and ATRT tumors are known to upregulate _MYC_ [@doi:10.3390/genes8040107] which in turn activates _E2F_ and S phase [@pmid:11511364].
+For example, HGG, DMG, MB, and ATRT tumors are known to upregulate _MYC_ [@doi:10.3390/genes8040107] which in turn activates _E2F_ and S phase [@pubmed:11511364].
 Indeed, we detected significant (Bonferroni-corrected p < 0.05) upregulation of _MYC_ and _E2F_ targets, as well as G2M (cell cycle phase following S phase) in MBs, ATRTs, and HGGs compared to several other cancer groups.
 In contrast, LGGs showed significant downregulation (Bonferroni-corrected p < 0.05) of these pathways.
```
Comment (jaclyn-taroni) [inline content/03.results.md:288]:
```suggestion
In contrast, LGGs showed significant downregulation (Bonferroni-corrected p < 0.05, multiple cancer group comparisons) of these pathways.
```

**Review by jaclyn-taroni — APPROVED** [review]
I can not easily make suggestions on the rest of the paragraph I called out here: https://github.com/AlexsLemonade/OpenPBTA-manuscript/issues/402#issuecomment-1447322317 and it seemed silly to file a PR stacked on this at this point.

So instead: if the addition in the one suggestion I _am able to return_ looks good to you, please propagate this to the sentence about cytokine pathways in schwannomas and neurofibromas.

**Pull request merged**